### ISSUE TITLE
[texture] background scrolling feels shaky / shaking

### ISSUE THREAD

**Issue by gengjun** [raysan5/raylib#4625]
When scrolling the background with the method in this example
https://www.raylib.com/examples/textures/loader.html?name=textures_background_scrolling

if you stare at the slow moving background in the back, you will see its shaky. once you noticed it, it's hard to unseen it. I think this is probably due to floating point rounding to integer and we can only shift the background by integers. This is also why fast moving background looks less shaking. I wonder if you have any suggestions  on how to eliminate this issue ? (i need to scroll some backgrounds slowly).

**Comment by raysan5** [comment]
Afaik, it is drawing using float positions so issue could be platform/os/gpu/drivers dependand and impossible to address on raylib side.

**Comment by gengjun** [comment]
tldr: the screen size has to be multiples of the images size
fyi: https://gamedev.stackexchange.com/questions/157011/strange-warping-shimmering-effect-with-unity-tilemap-and-cinemachine
